docs/source/accelerate/deepspeed.md

```diff
@@ -171,6 +173,115 @@ In the above example, the memory consumed per GPU is 64 GB (80%) as seen in the
 ## More resources
 You can also refer this blog post [Falcon 180B Finetuning using 🤗 PEFT and DeepSpeed](https://medium.com/@sourabmangrulkar/falcon-180b-finetuning-using-peft-and-deepspeed-b92643091d99) on how to finetune 180B Falcon model on 16 A100 GPUs on 2 machines.
 
+
+# Use PEFT QLoRA and DeepSpeed with ZeRO3 for finetuning large models on multiple GPUs
+
+In this section, we will look at how to use QLoRA and DeepSpeed Stage-3 for finetuning 70B llama model on 2X40GB GPUs.
+For this, we need to set `zero3_init_flag` to true when using Accleerate config. When passing `--deepspeed` argument with deepspeed stage 3 json config file path to `TrainingArguments` class, zero init is enabled by default. Here, we will be using accelerate config and below is the config which is also in configs folder at [deepspeed_config_z3_qlora.yaml](https://github.com/huggingface/peft/blob/main/examples/sft/configs/deepspeed_config_z3_qlora.yaml):
```

Comment:
The whole paragraph is a little confusing to me. I tried to break it up a bit, but I'm not sure if I captured the meaning entirely correctly. Please check if my suggestion makes sense:

```suggestion
For this, we need to set `zero3_init_flag` to true when using `accelerate config`. Here, we show the expected settings after calling it:
```

Then below this, where we currently have:

> Launch command is given below which is available at [run_peft_qlora_deepspeed_stage3.sh](https://github.com/huggingface/peft/blob/main/examples/sft/run_peft_deepspeed.sh):

Change it like this:

> Alternatively, we can use [this shell script](https://github.com/huggingface/peft/blob/main/examples/sft/run_peft_deepspeed.sh) in combination with the [provided DeepSpeed config file](https://github.com/huggingface/peft/blob/main/examples/sft/configs/deepspeed_config_z3_qlora.yaml). In this case, zero init is enabled by default.

Reply:
I will rewrite this, the above suggestion isn't what I am conveying.